.github/workflows/ci.yml

```diff
@@ -131,6 +131,51 @@ jobs:
       - name: Test
         run: make test
 
+  test-lint-migrations:
```

Comment:
Question: Does this need to be its own job? Could it be added to either `test-python` or `lint`?

Reply:
I suggest adding it as part of `lint`.

Reply:
The settings is different (test vs dev) but updated to `run: DJANGO_SETTINGS_MODULE=cms.settings.dev make lint-migrations` in [b22b9c0](https://github.com/ONSdigital/dis-wagtail/pull/43/commits/b22b9c03f24fddd3794286fd6b1c504513cbbcd0) credits to @zerolab